hudi-common/src/test/java/org/apache/hudi/common/model/TestOverwriteWithLatestAvroPayload.java

```diff
@@ -54,13 +57,15 @@ public void testActiveRecords() throws IOException {
     record1.put("id", "1");
     record1.put("partition", "partition0");
     record1.put("ts", 0L);
-    record1.put("_hoodie_is_deleted", false);
+    record1.put(defaultDeleteField, false);
+    record1.put(deleteField, true);
```

Comment:
shouldn't this also be false.

Reply:
In fact, it does not matter whether deleteField is true or false here, only defaultDeleteField will affect decide the result.